pkg/antctl/raw/multicluster/commands.go

```diff
@@ -49,6 +50,8 @@ var DeployCmd = &cobra.Command{
 	Short: "Deploy Antrea Multi-cluster Controller to a leader or member cluster",
 }
 
+var JoinCmd = join.NewJoinCommand()
```

Comment:
Besides join for member clusters, would you add a command for the leader or change "create clusterset" too? Pasted below what we discussed in #3921 :

> So, we just need two commands:
> 1. "deploy" to deploy MC controllers;
> 2. "create clusterset" (or "init" in leader and "join" in member as kubeadmin) to create/join a clusterset. For the leader, the command can generate a shared SA/token for member to join the clusterset.

Reply:
I meant we either:

1. change the current "create clusterset" command to create a clusterset & a default SA/taken in the leader; and create a clusterset &  join it in a member
2. or, add an "init" command to create a clusterset & a default SA/taken in the leader; and a "join" command to create a clusterset &  join it in a member

But if we go 2), how should a leader/member leave a ClsuterSet? Would you add anther "destroy" command for leader and a "leave" command for member?

In any case, we should remove the commands that are no longer valid, like "create claims", "add member", etc.

Reply:
Sure. I will add those commands. thanks for advice.

Reply:
Design added in https://github.com/antrea-io/antrea/issues/3921